### DISCUSSION TITLE
Basic GROUP BY

### DISCUSSION THREAD

**Discussion by dsaccon** [collerek/ormar#277] [General]
Hello, I'm new to ormar and I just want to do a simple GROUP BY type of query. E.g.

```
class Transactions(ormar.Model):
    class Meta(BaseMeta):
        tablename = "transactions"

    id: int = ormar.Integer(primary_key=True)
    tx_type: str = ormar.String(max_length=100)
    amount: float = ormar.Float(nullable=False)
```

`SELECT tx_type FROM transactions GROUP BY tx_type;`

Or even,

`SELECT tx_type, SUM (amount) FROM transactions GROUP BY tx_type;`

Is there a simple way to do this? Appreciate any examples to get me started

Thanks!

**Comment by collerek** [comment]
Hi as of now it's not possible, the enhancement with the same functionality is proposed: #266
